pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -271,9 +277,13 @@ void processArguments() throws Exception {
             }
             if (null != processingGuarantees) {
                 functionConfig.setProcessingGuarantees(processingGuarantees);
+            } else if (functionConfig.getProcessingGuarantees() == null) {
```

Comment:
this can be removed since proto3's conventions have a default value for enums

Reply:
sounds good